packages/ethereum-contracts/contracts/agreements/AgreementBase.sol

```diff
@@ -15,6 +14,8 @@ abstract contract AgreementBase is
 {
     address immutable internal _host;
 
+    error AgreementBase_OnlyHost();
```

Comment:
hmmm, we probably should just have one `OnlyHost`  error across, for example, move to a library "SuperfluidErrors" or even better if it could be defined as global instance.